src/script/api.ts

```diff
@@ -0,0 +1,37 @@
+const API_URL = 'https://dummyjson.com/posts';
+const LIMIT = 30;
+let skip = 0;
+let hasMore = true;
+let isFetching = false;
+
+export interface Post {
+    id: number;
+    title: string;
+    body: string;
+}
```

Comment:
Nit: can provide a meaningful name for interface.
```suggestion
export interface PostData {
    id: number;
    title: string;
    body: string;
}
```

Reply:
chnage https://github.com/Sakshi-Simform/product-management/pull/3/commits/d563ce1906f37594112f1f659420ebbaef655ce4